common/lib/xmodule/xmodule/library_content_module.py

```diff
@@ -313,6 +346,42 @@ def refresh_children(self, request, suffix, update_db=True):  # pylint: disable=
         lib_tools.update_children(self, user_id, user_perms, update_db)
         return Response()
 
+    def _validate_library_version(self, validation, lib_tools, version, library_key):
+        """
+        Validates library version
+        """
+        latest_version = lib_tools.get_library_version(library_key)
+        if latest_version is not None:
+            if version is None or version != latest_version:
+                validation.set_summary(
+                    StudioValidationMessage(
+                        StudioValidationMessage.WARNING,
+                        _(u'This component is out of date. The library has new content.'),
+                        # TODO: change this to action_runtime_event='...' once the unit page supports that feature.
+                        # See https://openedx.atlassian.net/browse/TNL-993
+                        action_class='library-update-btn',
+                        # Translators: ↻ is an UTF icon symbol, no need translating it.
```

Comment:
Since ↻ is no longer in the string that is scraped, translators won't see it. Therefore your comment doesn't really apply.

What WOULD be useful though is that you make the placeholder named and explain what will go there. For example, a placeholder named "refresh_icon" and a note for translators that refresh_icon is ↻. That way they can decide if the placement should be elsewhere relative to the text. As it is, they have no idea what will be substituted for {0}.